godotproject/enemy-Ant.tscn

```diff
@@ -1,75 +1,76 @@
-[gd_scene load_steps=20 format=2]
+[gd_scene load_steps=21 format=2]
 
 [ext_resource path="res://enemy-Ant.gd" type="Script" id=1]
 [ext_resource path="res://poison_frog.png" type="Texture" id=2]
+[ext_resource path="res://sfx/enemyFrogAttackSound.wav" type="AudioStream" id=3]
 
 [sub_resource type="AtlasTexture" id=1]
 atlas = ExtResource( 2 )
-region = Rect2( 0, 120, 40, 40 )
+region = Rect2( 0, 40, 40, 40 )
```

Comment:
Was this (and the similar other changes in the file) intentional? If so, what does it do?
My best guess is that Godot decided for some reason to swap around the order of the animations. I don't think it should have any functional impact.

Reply:
i have no idea what that was, ill uncheck it

Reply:
So I unchecked this but Godot just added it back again. Im not sure what it does but it must have some importance.